include/inexor/vulkan-renderer/gpu_info.hpp

```diff
@@ -2,18 +2,31 @@
 
 #include <vulkan/vulkan_core.h>
 
+#include <string>
+
 namespace inexor::vulkan_renderer {
 
 /// @brief Print information related to graphics card's capabilities and limits.
-struct VulkanGraphicsCardInfoViewer {
-
+class GPU_Info {
```

Comment:
Should probably be `GpuInfo` or at least not have the underscore.

Reply:
I renamed it to `GpuInfo`.

Reply:
Done.